ext/telemetry.c

```diff
@@ -32,6 +32,17 @@ void ddtrace_telemetry_finalize(void) {
 
     ddog_TelemetryActionsBuffer *buffer = ddog_sidecar_telemetry_buffer_alloc();
 
+    zend_module_entry *module;
+    char module_name[261] = { 'e', 'x', 't', '-' };
+    ZEND_HASH_FOREACH_PTR(&module_registry, module) {
```

Comment:
Should we report datadog extensions? I recall some discussions about this in other libraries, but I don't have a particularly strong opinion about this.  @Moutix what do you think?

Reply:
Yeah, this pretty much includes all modules, including ext-Core etc. But I'm not sure whether blacklisting a couple ones specifically makes a lot of sense?